src/components/nabla/common/NablaTokenPrice.tsx

```diff
@@ -5,14 +5,33 @@ export type TokenPriceProps = {
   address: string;
   prefix?: ReactNode;
   fallback?: ReactNode;
+  currentTokenAmount?: number;
```

Comment:
Let's generally avoid `number` for financial applications and only use the `Big.js` library or native `bigint`.

```suggestion
  currentTokenAmount?: Big;
```